From 4a643806f7416e5d98f72b9db0b906ec3ecef47f Mon Sep 17 00:00:00 2001
From: Josh Triplett <josh@joshtriplett.org>
Date: Mon, 23 Dec 2013 13:55:33 -0800
Subject: [PATCH 5/7] staging: wlags49_h2: Drop debug macro recording build
 date/time

Patch-mainline: 3.14-rc1
Git-commit: bf8c0f540cd480c327f9f8fe499c7c12a24526ac

The kernel already has this information, and individual drivers
shouldn't duplicate that.  This also eliminates the use of __DATE__ and
__TIME__, which make the build non-deterministic.

Signed-off-by: Josh Triplett <josh@joshtriplett.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Jan Moskyto Matejka <mq@suse.cz>
---
 drivers/staging/wlags49_h2/wl_version.h | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/drivers/staging/wlags49_h2/wl_version.h b/drivers/staging/wlags49_h2/wl_version.h
index 037b526..019ebb2 100644
--- a/drivers/staging/wlags49_h2/wl_version.h
+++ b/drivers/staging/wlags49_h2/wl_version.h
@@ -129,11 +129,7 @@ err: define bus type;
 #endif  /* HERMES25 */
 #endif  /* BUS_XXX */
 
-#ifdef DBG
-#define MODULE_DATE         __DATE__ " " __TIME__
-#else
 #define MODULE_DATE         "07/18/2004 13:30:00"
-#endif // DBG
 
 //#define STR2(m) #m
 //#define STR1(m) STR2(m)
-- 
1.8.4.5

